ntbs-service/Models/Entities/ClinicalDetails.cs

```diff
@@ -50,17 +50,21 @@ public class ClinicalDetails : ModelBase, IOwnedEntityForAuditing
         public int? BCGVaccinationYear { get; set; }
         public HIVTestStatus? HIVTestState { get; set; }
 
-        [OnlyOneTrue("IsMDRTreatment", ErrorMessage = ValidationMessages.ValidTreatmentOptions)]
```

Comment:
Was this the only place we used `OnlyOneTrueAttribute`? We could delete it if so!

Reply:
Sounds good, deleted!